source/code-generation.txt

```diff
@@ -0,0 +1,56 @@
+.. _rm-code-generation:
+
+===============
+Code Generation
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can assist you in converting both your SQL and 
+application code to MongoDB. By utilizing Relational Migrator's code 
+generation features, you can streamline the migration process and 
+save time and effort.
+
+Use Cases
+---------
+
+You can use the following features within the 
+:guilabel:`Code Generation` tab to convert your legacy code:
```

Comment:
[suggestion]

Maybe clarify "convert your legacy code to MongoDB syntax" (or similar)?

Reply:
Updated this to be a bit more specific:

```
You can use the following features in the :guilabel:`Code Generation` 
tab to convert your relational database SQL and application code 
to work with MongoDB:
``